packages/zoe/src/contractFacet/contractFacet.js

```diff
@@ -145,6 +145,35 @@ export function buildRootObject() {
       getBrandForIssuer: issuer =>
         issuerTable.getIssuerRecordByIssuer(issuer).brand,
       getAmountMath,
+
+      makeZCFMint: (keyword, mathHelperName = 'nat') => {
```

Comment:
`makeLocalMint`?

Reply:
I often use "localFoo" to mean a Foo that is here in this vat rather than possibly remote in some other vat. But I don't like "ZCF" here either. We need a new name.